Dockerfile

```diff
@@ -65,6 +65,9 @@ RUN echo AllowUsers ocrd >> /etc/ssh/sshd_config
 RUN echo "cd /data" >> /etc/profile
 RUN /usr/sbin/sshd -t # check the validity of the configuration file and sanity of the keys
 COPY *.sh /usr/bin/
+RUN cd /usr/bin; \
```

Comment:
```suggestion
RUN cd /usr/bin \
```

Reply:
that would not work – you need a cmd separator